### PR TITLE
qingmr, hbase/manual *：Add the qingmr and new features

### PR DESCRIPTION
**Pull request by molliezhang**
### What type of PR is this?
 
/Add Features

### Which issue(s) this PR fixes?

NA

### What this PR does /  why we need it ?

Summary:
1. Add qingmr in bigdata
2. Add the new features about deploy-filter
3. Fix some errors

### Special notes for your reviewer?
NA

/assign @pethu1110

### CODE REVIEW

File: content/bigdata/qingmr/manual/config_para/config_para_info.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: "简介"
```
Comment (pethu1110) [inline content/bigdata/qingmr/manual/config_para/config_para_info.md:2]: 标题不太合适，“参数介绍”这类的

File: content/bigdata/qingmr/manual/database_userguide/database_info.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: "简介"
```
Comment (pethu1110) [inline content/bigdata/qingmr/manual/database_userguide/database_info.md:2]: 标题不太合适，“参数介绍”这类的

File: content/bigdata/qingmr/manual/config_para/config_para_info.md (hunk 1 above)
Comment (molliezhang) [reply content/bigdata/qingmr/manual/config_para/config_para_info.md]: ok

File: content/bigdata/qingmr/manual/database_userguide/database_info.md (hunk 2 above)
Comment (molliezhang) [reply content/bigdata/qingmr/manual/database_userguide/database_info.md:2]: ok

**Review by pethu1110 — APPROVED** [review]

**Pull request merged**